app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Recipes</title>
+  <%= stylesheet_link_tag 'recipe_index', media: 'all', 'data-turbolinks-track': 'reload' %>
+</head>
+<body>
+  <div class="container">
+    <%= button_to "Add recipe", new_user_recipe_path(user_id: @user.id), method: :get, class: "btn" %>
+    <br />
+    <% if @user.recipes.empty? %>
+      <p>No recipes added yet</p>
+    <% else %>
+      <% @user.recipes.each do |recipe| %>
+        <div class="recipe">
+          <p><%= link_to recipe.name, user_recipe_path(user_id: recipe.user_id, id: recipe.id), class: "recipe-name" %></p>
+          <p class="recipe-description"><%= recipe.description %></p>
```

Comment:
- [ ] To match the given wireframe, please use flexbox to display the description and the recipe name on one line.

given|yours
---|---
![image](https://github.com/smunoz1988/recipe-on-rails/assets/91486791/1b37dfdd-2b97-4901-868a-b30f008efb37)|![image](https://github.com/smunoz1988/recipe-on-rails/assets/91486791/71294132-1b5c-490a-8831-bd87964cf9ca)

Reply:
@tugsem  

changes done!

![image](https://github.com/smunoz1988/recipe-on-rails/assets/64698693/53f315b5-25e5-4b7e-92c1-7298873a5e47)